Error when testing customized Volto module in addon

**Describe the bug**
When customizing the [FormValidation.jsx](https://github.com/plone/volto/blob/main/packages/volto/src/helpers/FormValidation/FormValidation.jsx) module and copying its tests to the addon, I receive the error below when executing the tests:
```javascript
FAIL src/addons/my.addon/src/customizations/volto/helpers/FormValidation/FormValidation.test.js
  ● Test suite failed to run

    node_modules/@plone/volto/src/helpers/FormValidation/validators.ts:1:35 - error TS7016: Could not find a declaration file for module './FormValidation'. '/home/user/git/app/node_modules/@plone/volto/src/helpers/FormValidation/FormValidation.jsx' implicitly has an 'any' type.

    1 import { validationMessage } from './FormValidation';
```

**To Reproduce**
Steps to reproduce the behavior:

1. Create a Volto app with an addon.
2. Create the file:

`app/src/addons/my.addon/src/customizations/volto/helpers/FormValidation/FormValidation.jsx`

And place the contents of the corresponding Volto file, replacing relative imports with absolute imports.
3. Create the file:

`app/src/addons/my.addon/src/customizations/volto/helpers/FormValidation/FormValidation.test.js`

And place the contents of the corresponding Volto file, replacing relative imports with absolute imports.
4. Run the command:
```
CI=true yarn test --no-cache
```

**Expected behavior**
The test should run without errors.

**Software (please complete the following information):**

Volto 18.0.0-alpha.42
Plone 6.0.12
plone.restapi 9.7.1
CMF 3.5
Zope 5.10
Python 3.12.3 (main, Apr 27 2024, 19:00:21) [GCC 11.4.0]
PIL 9.5.0 (Pillow)

**Additional context**
I tried creating a `FormValidation.d.ts` file but it didn't work.

--- a/packages/volto/news/6244.bugfix
+++ b/packages/volto/news/6244.bugfix
@@ -1,1 +1,1 @@
-Fixes imports to avoid type errors in addons tests. @wesleybl
+Changed imports from relative to absolute to avoid type errors in add-on tests. @wesleybl

--- a/packages/volto/news/6244.bugfix
+++ b/packages/volto/news/6244.bugfix
@@ -1,0 +1,1 @@
+Fixes imports to avoid type errors in addons tests. @wesleybl

--- a/packages/volto/src/helpers/Blocks/cloneBlocks.ts
+++ b/packages/volto/src/helpers/Blocks/cloneBlocks.ts
@@ -4,7 +4,7 @@ import {
   getBlocksFieldname,
   getBlocksLayoutFieldname,
   hasBlocksData,
-} from './Blocks';
+} from '@plone/volto/helpers/Blocks/Blocks';
 import config from '@plone/registry';
 
 export function cloneBlocks(blocksData) {

--- a/packages/volto/src/helpers/FormValidation/FormValidation.jsx
+++ b/packages/volto/src/helpers/FormValidation/FormValidation.jsx
@@ -1,5 +1,5 @@
 import { map, keys, intersection, isEmpty } from 'lodash';
-import { messages } from '../MessageLabels/MessageLabels';
+import { messages } from '@plone/volto/helpers/MessageLabels/MessageLabels';
 import config from '@plone/volto/registry';
 import { toast } from 'react-toastify';
 import Toast from '@plone/volto/components/manage/Toast/Toast';

--- a/packages/volto/src/helpers/FormValidation/validators.ts
+++ b/packages/volto/src/helpers/FormValidation/validators.ts
@@ -1,4 +1,4 @@
-import { validationMessage } from './FormValidation';
+import { validationMessage } from '@plone/volto/helpers/FormValidation/FormValidation';
 import { messages } from '@plone/volto/helpers/MessageLabels/MessageLabels';
 
 type MinMaxValidator = {

